var-core/src/main/java/io/varhttp/VarServlet.java

```diff
@@ -67,7 +69,12 @@ public void handle(HttpServletRequest request, HttpServletResponse response) {
 		}
 		Request r = new Request(httpMethod, servletPath);
 
-		final String[] requestPath = r.path.substring(1).split("/");
+		final String[] requestPath;
+		if(redirects.containsKey(r.path)) {
```

Comment:
space

Reply:
7c48ebf0

Reply:
I reformatted the entire file, IntelliJ disagree that there is anything wrong :P

Reply:
ah no im blind